docs/ref/checks.txt

```diff
@@ -237,6 +240,9 @@ Related Fields
 * **fields.W341**: ``ManyToManyField`` does not support ``validators``.
 * **fields.W342**: Setting ``unique=True`` on a ``ForeignKey`` has the same
   effect as using a ``OneToOneField``.
+* **fields.E350**: The field ``<app label>.<model>.<field name>`` was declared
```

Comment:
I noticed looking over the existing codes that they seemed vaguely grouped, with breaks like this introducing new groups of similar checks (E320, E330 etc). Looking over it again it seems this check perhaps belongs with the first group of E300s.